public/src/Components/ErrorPage.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { JsxElement } from 'typescript';
+import {
+    BrowserRouter as Router,
+    Route,
+    Link,
+    Switch,
+    Redirect,
+  } from "react-router-dom";
```

Comment:
why do you use double and single quotes? choose one option and use it through all code